website/events/payables.py

```diff
@@ -24,12 +25,17 @@ def payment_notes(self):
     def payment_payer(self):
         return self.model.member
 
-    def can_create_payment(self, member):
-        return is_organiser(member, self.model.event)
+    def can_manage_payment(self, member):
+        return is_organiser(member, self.model.event) and member.has_perm(
+            "events.change_eventregistration"
+        )
 
     @property
     def tpay_allowed(self):
-        return self.model.event.tpay_allowed
```

Comment:
I think the purpose of this property is to see whether tpay payments are allowed for the whole payable. The check for whether the user actually _can_ do a tpay payment is implemented in the payments app itself